src/orquestra/sdk/_base/abc.py

```diff
@@ -93,7 +94,7 @@ def get_workflow_run_status(self, workflow_run_id: WorkflowRunId) -> WorkflowRun
     @abstractmethod
     def get_workflow_run_outputs_non_blocking(
         self, workflow_run_id: WorkflowRunId
-    ) -> t.Sequence[ArtifactValue]:
+    ) -> t.Sequence[WorkflowResult]:
```

Comment:
@alexjuda `get_workflow_run_outputs_non_blocking` returns a tuple of `WorkflowResults`, not singular.

The reason for this is because we may have this scenario:
```
@sdk.workflow
def wf():
    a = t1()
    b = t2(a)
    c,d = t3()
    _, e = t4(a,c)
    return a,b,c,d,e
```

We have all these values, but we don't want to deserialize from these values just to pack them in a single value. This is the problem that we're trying to solve because of the missing deps.